site/src/cases/data.ts

```diff
@@ -0,0 +1,60 @@
+import { client } from '../api';
+import { CmsCasesModel, cmsCasesToSite, SiteCasesModel } from './types';
+
+export interface SiteCasesData {
+  cases: SiteCasesModel;
+}
+
+export async function getCasesData(): Promise<SiteCasesData> {
+  const {
+    data: { cases },
+  } = (await client({
+    query: `query {
+  cases {
+    id
+    header {
+      title {
+        en
+      }
+      tagline {
+        en
+      }
+      content {
+        en
+      }
+      image
+      client
+      clientwebsite
+      fotocredits
+    }
+    content {
+      id
+      image {
+        alternativeText
+        width
+        height
+        url
+        caption
+      }
+      title {
+        en
+      }
+      tagline {
+        en
+      }
+      content {
+        en
+      }
+      cta {
+        href
+        text {
+          en
+        }
+      }
+    }
+  }
+}
+`,
+  })) as { data: { cases?: CmsCasesModel } };
+  return { cases: cmsCasesToSite(cases) };
+}
```

Comment:
i'm having problem here it seems there is a mistake but i can't find what is wrong with this setup. Fyi i already copied the query and test it in graphql and it seems fine